lib/fhir_models/fhir/resources/AuditEvent.rb

```diff
@@ -36,15 +36,15 @@ class Agent < FHIR::Model
'122670', '122675', '122680', '122683', '122684', '122685', '122686', '122687', '122698', '122699', '122700', '122701', '122702', '122703', '122704', '122705', '122706', '122707', '122708', '122709', '122710', '122711', '122712', '122713', '122715', '122716', '122717', '122718', '122720', '122721', '122726', '122727', '122728', '122729', '122730', '122731', '122732', '122733', '122734', '122735', '122739', '122740', '122741', '122742', '122743', '122744', '122745', '122748', '122750', '122751', '122752', '122753', '122755', '122756', '122757', '122758', '122759', '122760', '122762', '122764', '122768', '122769', '122770', '122771', '122772', '122773', '122775', '122776', '122781', '122782', '122783', '122784', '122785', '122791', '122792', '122793', '122795', '122796', '122797', '122799', '123001', '123003', '123004', '123005', '123006', '123007', '123009', '123010', '123011', '123012', '123014', '123015', '123016', '123019', '123101', '123102', '123103', '123104', '123105', '123106', '123107', '123108', '123109', '123110', '123111', '125000', '125001', '125002', '125003', '125004', '125005', '125006', '125007', '125008', '125009', '125010', '125011', '125012', '125013', '125015', '125016', '125021', '125022', '125023', '125024', '125025', '125030', '125031', '125032', '125033', '125034', '125035', '125036', '125037', '125038', '125040', '125041', '125100', '125101', '125102', '125105', '125106', '125107', '125195', '125196', '125197', '125200', '125201', '125202', '125203', '125204', '125205', '125206', '125207', '125208', '125209', '125210', '125211', '125212', '125213', '125214', '125215', '125216', '125217', '125218', '125219', '125220', '125221', '125222', '125223', '125224', '125225', '125226', '125227', '125228', '125230', '125231', '125233', '125234', '125235', '125236', '125237', '125238', '125239', '125240', '125241', '125242', '125251', '125252', '125253', '125254', '125255', '125256', '125257', '125258', '125259', '125261', '125262', '125263', '125264', '125265', '125270', '125271', '125272', '125273', '125901', '125902', '125903', '125904', '125905', '125906', '125907', '125908', '126000', '126001', '126002', '126003', '126010', '126011', '126020', '126021', '126022', '126030', '126031', '126032', '126033', '126034', '126035', '126036', '126037', '126038', '126039', '126040', '126050', '126051', '126052', '126060', '126061', '126062', '126063', '126064', '126065', '126066', '126067', '126070', '126071', '126072', '126073', '126074', '126075', '126080', '126081', '126100', '126200', '126201', '126202', '126203', '126220', '126300', '126301', '126302', '126303', '126310', '126311', '126312', '126313', '126314', '126320', '126321', '126322', '126330', '126331', '126340', '126341', '126342', '126343', '126344', '126350', '126351', '126352', '126353', '126360', '126361', '126362', '126363', '126364', '126370', '126371', '126372', '126373', '126374', '126375', '126376', '126377', '126380', '126390', '126391', '126392', '126393', '126394', 
```

Comment:
Interesting thing here.  `_CitizenRoleType` is the only code in this CodeSystem and relevant ValueSet that begins with an underscore that is not abstract.  What's up with that?  Does the underscore have any meaning?

[ValueSet](https://www.hl7.org/fhir/valueset-security-role-type.html)
[CodeSystem](https://www.hl7.org/fhir/v3/RoleCode/cs.html#v3-RoleCode-_CitizenRoleType)

Reply:
My guess is that `_CitizenRoleType` was meant to be abstract, but wasn't marked appropriately. It is the same in `4.0.1`. It is a code imported from v3, so if it is a mistake or bug, it might have to be fixed there first.